setup.py

```diff
@@ -162,10 +162,13 @@ def fill_connectors(projects, modules_path):
         },
     }
 
-    # Prepare MANIFEST.in, include json files
+    # Prepare MANIFEST.in, include json files, utitls
     shutil.rmtree('MANIFEST.in', ignore_errors=True)
     shutil.copyfile('build_templates/MANIFEST.in', 'MANIFEST.in')
     json_include_lines = []
+    utils_include_list = {
+        'stix_shifter_utils/stix_translation/src/json_to_stix/json_to_stix_translator.py': '%s/stix_translation/json_to_stix_translator.py'
```

Comment:
does this mean only `json_to_stix_translator.py` file included in the util package?

Reply:
I have a feeling that it may include also [observable.py](https://github.com/opencybersecurityalliance/stix-shifter/blob/SECCON-12394-Package-json_to_stix_translator-into-connector-whl/stix_shifter_utils/stix_translation/src/json_to_stix/observable.py) in the future

Reply:
I think we need to discuss about this with @yurii-klymenko